.gitignore

```diff
@@ -8,6 +8,7 @@ __pycache__/
 .develop/
 .cache/
 .pytest_cache/
+.vscode/
```

Comment:
It's not a good practice to put your personal development environment related paths to project's `.gitignore` file. Please configure global git exclude file on your development environment (`$HOME/.config/git/ignore`). See `man gitignore` for more details.